File: atelier_couture/src/data/entites/ArticleVente.java
package data.entites;

public class ArticleVente extends Article{

    private int prixVente;
    private int montVente;
    public ArticleVente() {}

    public int getPrixVente() {
        return prixVente;
    }

    public void setPrixVente(int prixVente) {
        this.prixVente = prixVente;
    }

    public int getMontVente() {
        return montVente;
    }

    public void setMontVente(int montVente) {
        this.montVente = montVente;
    }

    @Override
    public String toString() {
        return "ArticleVente{" +
                "id=" + id +
                ", categorie=" + categorie +
                ", photo='" + photo + '\'' +
                ", qteStock=" + qteStock +
                ", libelle='" + libelle + '\'' +
                ", montVente=" + montVente +
                ", prixVente=" + prixVente +
                '}';
    }
}

File: model/ArticleVente.php
<?php

require_once __DIR__ . '/Article.php';

class ArticleVente extends Article{
    private int $prixVente;
    private int $montVente;


    public function getPrixVente()
    {
        return $this->prixVente;
    }


    public function setPrixVente($prixVente)
    {
        $this->prixVente = $prixVente;

        return $this;
    }


    public function getMontVente()
    {
        return $this->montVente;
    }


    public function setMontVente($montVente)
    {
        $this->montVente = $montVente;

        return $this;
    }


    public static function toArticleVente($row):ArticleVente{
        $articleVente=new ArticleVente();
        $articleVente->setId($row['id']);
        $articleVente->setLibelle($row['libelle']);
        $articleVente->setPrixVente($row['prixVente']);
        $articleVente->setQteStock($row['qteStock']);
        $articleVente->setMontVente($row['montStock']);
        $articleVente->setCategorie($row['categorie']);
        return $articleVente;

    }

}